### PR TITLE
Fix failing case of block quotes by adding endWithOpenBlockOrParagraph

### PR DESCRIPTION
**Pull request by LutSa**
The issue of block quote https://github.com/dillonkearns/elm-markdown/issues/96 is caused by unimplemented open paragraph rule in block quote. The function endWithOpenBlockOrParagraph to judge whether or not a block quote is ended with open paragraph.

### CODE REVIEW

File: src/Markdown/Parser.elm [hunk 1]
```diff
@@ -684,6 +684,34 @@ parseAllInlinesHelp state rawBlocks parsedBlocks =
...
+        case blockslist of 
+            blocks :: _ ->
+                case blocks of
+                    last :: _ ->
+                        endWithOpenBlockOrParagraph last
+                    _ -> False
+            _ -> False
+    _  -> False
+
+    --| UnorderedListBlock Bool Int (List CloseListItem) OpenListItem
```
Comment (LutSa) [inline src/Markdown/Parser.elm:708]: Here are some cases might needed to add into that function. I might look at these cases after I get a closer look at the spec

**Comment by dillonkearns** [comment]
Fantastic, looks like great progress! Thank you for the PR!

Let me know when you're ready for a review, so far it's looking good 👍

**Comment by LutSa** [comment]
Yes, this PR is ready for a review.

**Comment by dillonkearns** [comment]
Looks great, thank you so much for the PR @LutSa! 🙏  Happy New Year!

**Comment by dillonkearns** [comment]
Published in version 7.0.1!

**Pull request merged**